reconcile/change_owners/change_log_tracking.py

```diff
@@ -134,8 +134,12 @@ def run(
                 continue
             diff = QontractServerDiff(**obj)
             changes = aggregate_resource_changes(
-                aggregate_file_moves(parse_bundle_changes(diff)),
-                {c.path: c.dict() for c in namespaces + jenkins_configs},
+                bundle_changes=aggregate_file_moves(parse_bundle_changes(diff)),
+                content_store={c.path: c.dict() for c in namespaces + jenkins_configs},
+                supported_schemas={
+                    "/openshift/namespace-1.yml",
+                    "/dependencies/jenkins-config-1.yml",
+                },
```

Comment:
can we extract this to a const and doc what schema should be supported and what shouldn't?

Reply:
i'll follow up in the future